osquery/tables/system/BUCK

```diff
@@ -172,6 +172,7 @@ osquery_cxx_library(
                 "darwin/crashes.cpp",
                 "darwin/cups_destinations.cpp",
                 "darwin/cups_jobs.cpp",
+		"darwin/diagnostics.cpp",

```

Comment:
whitespace looks off. `clang-format` perhaps?

Reply:
diagnostics is a separate pull request #5562, accidentally got included in this branch
Diagnostic related files should be removed now
Formatting should be fixed in other pull request